app/account/page.jsx

```diff
@@ -0,0 +1,109 @@
+'use client'
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react'
+import { useUserContext } from '../context/usercontext'
+import { useRouter } from 'next/navigation'
+import Profile from '../components/profile'
+
+function CustomTabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+CustomTabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.number.isRequired,
+  value: PropTypes.number.isRequired,
+};
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+export default function BasicTabs() {
+    const [value, setValue] = React.useState(0);
+    
+    const {user, setUser, userData, setUserData} = useUserContext();
+    const [dataLoaded, setDataLoaded] = useState(false);
+    let prevUser = null;
+    const router = useRouter();
+
+    useEffect(()=>{
+        if (user===null) {
+        try {
+            prevUser = JSON.parse(sessionStorage.getItem('firebase:authUser:AIzaSyCoGURJeUWdIylWkAEDYEpOqY6YnAaJYy0:[DEFAULT]'))
+            setUser(prevUser)
+        } catch {}
```

Comment:
if there is no catch logic here, what is the purpose of having a `try/catch`? Do you want to add an error use case here? If so, you should add a `TO-DO` type comment to say I'm coming back to fix this later. otherwise this code just looks strange.